AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AssertAdminAdobeStockImageAttributesLocalizedActionGroup.xml

```diff
@@ -7,14 +7,14 @@
 -->
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AssertAdminAdobeStockKeywordLocalizedActionGroup">
+    <actionGroup name="AssertAdminAdobeStockImageAttributesLocalizedActionGroup">
```

Comment:
Let's revert the unnecessary renaming of the action groups / steps to avoid potential conflicts